pkg/apis/contrail/v1alpha1/base_types.go

```diff
@@ -100,6 +100,12 @@ type PodConfiguration struct {
 	// +k8s:conversion-gen=false
 	// +optional
 	HostNetwork *bool `json:"hostNetwork,omitempty" protobuf:"varint,11,opt,name=hostNetwork"`
+	// HostAliases is an optional list of hosts and IPs that will be injected into the pod's hosts
+	// file if specified. This is only valid for non-hostNetwork pods.
```

Comment:
non-hostNetwork pods?

Reply:
It looks like this comment is not true. It was copy passed from core/v1/types.go.
However this is supported for hostnetworking:
https://github.com/kubernetes/kubernetes/commit/4edd92f26d812c2bc7b7442e6a553ca3f60a4a8e

Reply:
I will delete that sentence from this comment

Reply:
done